lib/post_action_creator.rb

```diff
@@ -114,7 +114,8 @@ def perform
     end
 
     # create meta topic / post if needed
-    if @message.present? && %i[notify_moderators notify_user spam].include?(@post_action_name)
+    if @message.present? &&
+         %i[notify_moderators notify_user spam illegal].include?(@post_action_name)
```

Comment:
Potential enhancement would be to lift this into the `PostActionType` setup so there'll only be a single place to add new flag types.

Reply:
Yes that would be much nicer, maybe something like `PostAction.creates_meta_post?`?